recipe/0303-Fixed-errors-encountered-with-GCC-11.patch

```diff
@@ -0,0 +1,84 @@
+From 19a40f09c743b638ddfa810886a812363414b583 Mon Sep 17 00:00:00 2001
+From: Nishidha Panpaliya <npanpa23@in.ibm.com>
+Date: Wed, 15 Jun 2022 14:09:53 +0000
+Subject: [PATCH] Fixed errors encountered with GCC 11
+
+---
+ dali/pipeline/executor/executor_test.cc       | 12 ++++++------
+ dali/pipeline/operator/sequence_shape_test.cc |  4 ++--
+ include/dali/core/convert.h                   |  2 +-
+ 3 files changed, 9 insertions(+), 9 deletions(-)
+
+diff --git a/dali/pipeline/executor/executor_test.cc b/dali/pipeline/executor/executor_test.cc
+index d914054f..2c7aaaeb 100644
+--- a/dali/pipeline/executor/executor_test.cc
++++ b/dali/pipeline/executor/executor_test.cc
+@@ -365,24 +365,24 @@ TYPED_TEST(ExecutorTest, DISABLED_TestDataSetup) {
+     ASSERT_EQ(hws.NumInput(), 0);
+     ASSERT_EQ(hws.NumOutput(), 1);
+     ASSERT_EQ(hws.GetRequestedBatchSize(0), this->batch_size_);
+-    ASSERT_TRUE(hws.OutputIsType<CPUBackend>(0));
++    //ASSERT_TRUE(hws.OutputIsType<CPUBackend>(0));
```

Comment:
@JanuszL - I'd to comment these lines from executor_test as these lines were giving error - `"error: 'this' pointer is null" Wnon-null`

Reply:
Interesting. If that helps you feel free. We will try to look into it in spare time but I cannot promise any ETA.

Reply:
Sure @JanuszL . No problem. Since it's a test, we are safe to have this commented.